free-3.md

```diff
@@ -4,51 +4,71 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+I'd like to see if it is possible to extend the work of Milo Han from last semester. He (or she?) wrote a program for sight reading. The language would serve music teachers and their students in improving their sight reading. A language might be good as it would be able to specify what kind of music one wants to sight read in order to sharpen one's weaknesses.
```

Comment:
Haha I know this guy. Sent him a screenshot of this and he had a good laugh at the he or she part.